.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: Backend CI
+
+on:
+  push:
+    branches-ignore:
+      - 'master'
+    tags-ignore:
+      - '**'
+    paths:
+      - '**.php'
+      - '**.yml'
+      - '**.yaml'
+      - 'app/schema/**.xml'
+      - 'composer.lock'
+      - 'phpcs.xml.dist'
+      - 'phpunit.xml.dist'
+      - 'psalm.xml'
+      - 'psalm-baseline.xml'
+
+jobs:
+  test:
+    name: Test
+
+    runs-on: ubuntu-18.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - '7.2'
+          - '7.3'
+          - '7.4'
+          - '8.0'
+
+    env:
+      APP_ENV: prod
+      APP_DEBUG: 0
+      COMPOSER_TOKEN: ${{ secrets.MOS_EU_GITHUB_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Setup Environment
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version}}
+          coverage: none
+          extensions: bcmath, exif, gd, gmp, imagick, intl, mysqli, opcache, pcntl, pdo_mysql, redis, soap, sockets, tidy, zip
```

Comment:
Why? according to the `composer.json` there are not requirements for any extension